Meta/refresh-serenity-qtcreator.sh

```diff
@@ -11,4 +11,4 @@ fi
 
 cd "$SERENITY_SOURCE_DIR"
 
-find . \( -name Base -o -name Patches -o -name Ports -o -name Root -o -name Toolchain -o -name Build \) -prune -o \( -name '*.ipc' -or -name '*.cpp' -or -name '*.idl' -or -name '*.c' -or -name '*.h' -or -name '*.S' -or -name '*.css' -or -name '*.cmake' -or -name '*.json' -or -name '*.gml' -or -name 'CMakeLists.txt' \) -print > serenity.files
+find . \( -name Base -o -name Patches -o -name Ports -o -name Root -o -name Toolchain -o -name Build \) -prune -o \( -name '*.ipc' -or -name '*.cpp' -or -name '*.idl' -or -name '*.c' -or -name '*.h' -or -name '*.in' -or -name '*.S' -or -name '*.css' -or -name '*.cmake' -or -name '*.json' -or -name '*.gml' -or -name 'CMakeLists.txt' \) -print > serenity.files
```

Comment:
These changes seem fine, but ... can QtCreator understand `compile_commands.json` yet? 😅 . Maybe there's something else about our CMake setup that makes us have to write this massive files list though.

Reply:
I'm not sure how that could be done, if you have suggestions I'd be happy to hear them ^^

Reply:
All that I can find is a [six-year old dead forum thread](https://forum.qt.io/topic/79805/qtcreator-clang-code-model-with-compile_commands-json) where it was suggested and (apparently) never followed-up.